src/css/tooltip.scss

```diff
@@ -0,0 +1,73 @@
+.tooltip {
+    position: relative;
+    display: inline-block;
+
+    .tooltip-content {
+        display: none;
+        background-color: $black;
+        padding: 6px;
+        border-radius: $grid-base;
+        color: $white;
+        @extend .h5;
+
+        position: absolute;
+        bottom: calc(100% + #{$grid-base});
+        left: 50%;
+        transform: translateX(-50%);
+        width: max-content;
+
+        &.visible {
+            display: block;
+        }
```

Comment:
This can go 🙅‍♂️ now with line 70 in the mix.

Let's handle it in the next PR though.